index.rst

```diff
@@ -269,6 +269,64 @@ our toolkit in the future (see how you can :ref:`get involved <contribute>`).
    </div>
 
 
+
+.. _started:
+
+Getting Started
+===============
+
+
+In order to start using the Fatiando tools you need to install a Python distribution. We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`__. You can find detailed instructions on how to install it on any operating system in the `Software Carpentry setup instructions <https://carpentries.github.io/workshop-template/#python>`__.
+
+Once you finished setting up Anaconda, you can install all the latest Fatiando tools with a single command:
+
+.. raw:: html
+
+   <code>
+      conda install verde harmonica rockhound boule --channel conda-forge
+   </code>
```

Comment:
```suggestion
.. code::

    conda install verde harmonica rockhound boule pooch --channel conda-forge
```
Using the rst block for code is probably better since it already includes formatting and padding etc. Also, I know pooch gets installed with the other tools but this way it's more complete 🙂